README.asciidoc

```diff
@@ -389,6 +389,67 @@ kak-lsp supports showing diagnostics inline after their respective line, but thi
 lsp-inlay-diagnostics-enable global
 ----
 
+== Markdown parsing
+
+kak-lsp offers multiple ways to show additional information provided by the language server, such as documentation for the token under the cursor or documentation for completion candidates. The Language Server Protocol allows for both plain text and Markdown in these scenarios, and most servers do implement Markdown.
+
+To utilize that, kak-lsp parses the Markdown into Kakoune's markup language, utilizing various faces for styling.
+
+To ensure compatiblity, these faces are, by default, undefined and fall back to whatever the regular styling is in the environment they are used (such as the `Information` face in an `info` box). To enable Markdown highlighting, define the following faces in your theme or `kakrc`:
+
+[cols="1a,3a"]
+|===
+| Face | Usage
+
+| `InfoDefault`
+| The default text color. You'll likely want to set this to `Information`, the default face for text in the info box.
+Leaving it blank has the same effect.
+
+| `InfoLink`
+| The face used to highlight link titles. Maybe some classic `blue+u` for this one?
+
+| `InfoMono`
+| The face used for inline monospace blocks, i.e. inline code.
+
+| `InfoLinkMono`
+| This face is assigned to inline code within link titles, such as in the following Markdown snippet. Here `format` would receive the `InfoLinkMono` face.
+
+----
+[the `format` function](https://example.com)
+----
+
+| `InfoBlock`
+| The face used for code blocks. Language specific syntax highlighting for code blocks is not possible.
+
+| `InfoBullet`
+| The face used to highlight the list symbol for both ordered and unordered lists. For the list item's text `InfoDefault` is used.
+
+| `InfoHeader`
+| The face used for headings. There is currently no distinction between different heading levels.
+
+| `InfoRule`
+| The face used for horizontal lines (rules).
```

Comment:
> I wonder if there is a realistic chance that someone actually wants to configure this to be different from InfoDefault?

Definitely non-zero. My colorscheme includes
```kak
face global InfoDefault               ${bg},${fg}
face global InfoRule                  ${bg4}
```

> it might be nice to render rules as continuous lines, just like the border of the info box.

That would certainly be nice, but we would need to know the exact size of the info box we end up rendering into. And as far as I am aware, there is no way for us to get that information.